onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -165,63 +173,63 @@ void SetIODefs(const ONNX_NAMESPACE::ModelProto& model_proto,
   }
 }
 
-std::vector<OrtValue*>
-GetOutputTensors(Ort::CustomOpApi& ort, OrtKernelContext* context, size_t batch_size,
+OrtValue*
```

Comment:
Why is this not guarded by #if defined's  like the other GetOutputTensor version?

Reply:
Nevermind.. i understand now

Reply:
The second GetOutputTensor is only for 2020.4 and higher OpenVINO versions because of constant folding. This GetOutputTensor method should be available for all openvino versions